actions/create_pull.py

```diff
@@ -0,0 +1,18 @@
+from lib.base import BaseGithubAction
+from github import GithubObject
+from lib.formatters import pull_to_dict
+
+__all__ = [
+    'CreatePullAction'
+]
+
+
+class CreatePullAction(BaseGithubAction):
+    def run(self, user, repo, title, head, base, body=None,
+            maintainer_can_modify=None):
+        user = self._client.get_user(user)
+        repo = user.get_repo(repo)
+        pull = repo.create_pull(title=title, head=head, base=base, body=body or '',
+                                maintainer_can_modify=maintainer_can_modify or GithubObject.NotSet)
```

Comment:
Is the `issue` parameter optional?  Can you add it as an optional parameter of the action?

Reply:
I didn't include that `issue` parameter, because I couldn't find it in the [GitHub API](https://developer.github.com/v3/pulls/#create-a-pull-request).

The PyGithub library expects you to either specify
```python
def __create_pull_1(
    self, title, body, base, head, maintainer_can_modify=github.GithubObject.NotSet
):
```
(https://github.com/PyGithub/PyGithub/blob/v1.45/github/Repository.py#L1312)

or

```python
def __create_pull_2(self, issue, base, head):
```
(https://github.com/PyGithub/PyGithub/blob/v1.45/github/Repository.py#L1332)

So, my code is using the `__create_pull_1` parameters.